extras/check_code_style.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+
+# static code analysis
+START_SHA=master
+END_SHA=HEAD
+
+NEST_SRC=.
+CPPCHECK=cppcheck
+VERA=vera++
+VERA_HOME=/usr/local/lib/vera++
```

Comment:
`VERA_HOME` is not needed, and it would be wrong if `vera++` were installed from MacPorts. One could try to infer it from `which vera++`.